xbmc/rendering/dx/RenderSystemDX.cpp

```diff
@@ -659,7 +660,8 @@ bool CRenderSystemDX::CreateDevice()
     CLog::Log(LOGDEBUG, "%s - on adapter %S (VendorId: %#x DeviceId: %#x) with feature level %#x.", __FUNCTION__, 
                         m_adapterDesc.Description, m_adapterDesc.VendorId, m_adapterDesc.DeviceId, m_featureLevel);
 
-    m_RenderRenderer = StringUtils::Format("%S", m_adapterDesc.Description);
+    std::wstring renderer = StringUtils::Format(L"%s", m_adapterDesc.Description);
+    g_charsetConverter.wToUTF8(renderer, m_RenderRenderer);
```

Comment:
I'd prefer the use of the FromW/ToW methods in `platform/win32/CharsetConverter.h` as it makes the code easier and simpler with less temporary variables. Not a big deal though.